tachyon/zk/plonk/circuit/examples/simple_lookup_circuit.h

```diff
@@ -25,6 +25,10 @@ class SimpleLookupConfig {
                      const AdviceColumnKey& advice)
       : selector_(selector), table_(table), advice_(advice) {}
 
+  SimpleLookupConfig Clone() const {
+    return SimpleLookupConfig(selector_, table_, advice_);
+  }
+
```

Comment:
Please remove this.

Reply:
A circuit config class should have `Clone()`, since `FloorPlanner` consume a config of the circuit.
And this change will be made in a separate, exclusive commit!